consensus/consensus_service.go

```diff
@@ -39,20 +39,21 @@ func (consensus *Consensus) WaitForNewRandomness() {
 }
 
 // GetNextRnd returns the oldest available randomness along with the hash of the block there randomness preimage is committed.
-func (consensus *Consensus) GetNextRnd() ([32]byte, [32]byte, error) {
+func (consensus *Consensus) GetNextRnd() ([516]byte, [32]byte, error) {
 	if len(consensus.pendingRnds) == 0 {
-		return [32]byte{}, [32]byte{}, errors.New("No available randomness")
+		return [516]byte{}, [32]byte{}, errors.New("No available randomness")
```

Comment:
use constant to replace 516

Reply:
about the constant vs numbers. I thought it is purposely done this way (golang?) because most code in harmony use numbers for things like size of pub key,  size of signature etc... so I follow their style. 

if using constant, should I we also change those signatures/pub key?

Reply:
used constants to replace numbers